Plugins/Statistics/GitStatistics/FormGitStatistics.cs

```diff
@@ -160,7 +157,7 @@ private void InitializeLinesOfCode()
             _lineCounter = new LineCounter();
             _lineCounter.LinesOfCodeUpdated += lineCounter_LinesOfCodeUpdated;
 
-            Task.Factory.StartNew(LoadLinesOfCode);
+            Task.Run(() => LoadLinesOfCode());
```

Comment:
ThreadHelper.JoinableTaskFactory.RunAsync(() => Task.Run(() => LoadLinesOfCode()));

Reply:
@jbialobr  Is the point of that to be that we kick off threadpool work that is tracked by the JoinableTaskCollection so it blocks shutdown? If so, that works. My team usually writes it out as:
```cs
ThreadHelper.JoinableTaskFactory.RunAsync(() => LoadLinesOfCodeAsync());

async Task LoadLinesOfCodeAsync()
{
   await TaskScheduler.Default;
   // do stuff
}
```

This way we're separating concerns (of avoiding the UI thread for heavy/IO work vs. the fact that we won't be awaiting it but want to track it anyway).And each bit of code is simpler.

Reply:
Thanks, yes it should look exactly as you wrote, but if I am not mistaken it is beyond of the scope of this PR.

Reply:
➡️ Added this one to #4709 (there are other places in the code facing the same issue)